skops/io/_cli.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import annotations
+
+import argparse
+import logging
+import os
+import pathlib
+import pickle as pkl
+from typing import Optional
+
+from skops.io import dumps, get_untrusted_types
+
+
+def _convert(
+    input_file: os.PathLike, output_dir: pathlib.Path, is_trusted: bool = False
+):
+    model_name = pathlib.Path(input_file).stem
+
+    logging.info(f"Converting {model_name}")
+
+    with open(input_file, "rb") as f:
+        obj = pkl.load(f)
+    skops_dump = dumps(obj)
+
+    untrusted_types = get_untrusted_types(data=skops_dump)
+
+    if not untrusted_types:
+        logging.debug(f"No unsafe types found in {model_name}.")
+    else:
+        untrusted_str = "\n".join(untrusted_types)
+
+        logging.warning(
+            "Unsafe Types Detected!\n"
```

Comment:
```suggestion
            "Unknown Types Detected!\n"
```

I think @BenjaminBossan agrees here.